src/test/java/xyz/vegaone/easytrackingv2/userStory/UserStoryServiceTest.java

```diff
@@ -0,0 +1,111 @@
+import xyz.vegaone.easytrackingv2.service.UserStoryService;
+
+@SpringBootTest
+@RunWith(SpringRunner.class)
+public class UserStoryServiceTest {
+
+    private static final String USER_STORY_NAME = "user story name";
+    private static final String NEW_USER_STORY_NAME = "New user story name";
+    private static final String USER_STORY_DESCRIPTION = "user story name";
+    private static final String USER_STORY_STATUS = "user story name";
+    private static final Long USER_STORY_PRIORITY = 1L;
+    private static final Long USER_STORY_ESTIMATION = 2L;
+
+    @Autowired
+    private UserStoryService userStoryService;
+
+    @Test
+    public void createUserStoryTest() {
+        //given
+
+        //when
+        UserStory savedUserStory = buildAndSaveUserStory();
+
+        //then
+        Assert.assertNotNull("There should have been one user story saved in the database", savedUserStory);
+        Assert.assertEquals("The user story name should have matched", USER_STORY_NAME, savedUserStory.getName());
+        Assert.assertEquals("The user story description should have matched", USER_STORY_DESCRIPTION, savedUserStory.getDescription());
+        Assert.assertEquals("The user story status should have matched", USER_STORY_STATUS, savedUserStory.getStatus());
+        Assert.assertEquals("The user story priority should have matched", USER_STORY_PRIORITY, savedUserStory.getPriority());
+        Assert.assertEquals("The user story estimation should have matched", USER_STORY_ESTIMATION, savedUserStory.getEstimation());
+
+
+    }
+
+    @Test
+    public void getUserStory() {
+        //given
+        UserStory savedUserStory = buildAndSaveUserStory();
+
+        //when
+        UserStory findUserStory = userStoryService.getUserStory(savedUserStory.getId());
+
+        //then
+        Assert.assertNotNull("There should have been one user story saved in the database", savedUserStory);
+        Assert.assertEquals("The user story name should have matched", findUserStory.getName(), savedUserStory.getName());
+        Assert.assertEquals("The user story description should have matched", findUserStory.getDescription(), savedUserStory.getDescription());
+        Assert.assertEquals("The user story status should have matched", findUserStory.getStatus(), savedUserStory.getStatus());
+        Assert.assertEquals("The user story priority should have matched", findUserStory.getPriority(), savedUserStory.getPriority());
+        Assert.assertEquals("The user story estimation should have matched", findUserStory.getEstimation(), savedUserStory.getEstimation());
+
+    }
+
+    @Test
+    public void deleteUserStoryTest() {
+        //given
+        UserStory savedUserStory = buildAndSaveUserStory();
+
+        //when
+        userStoryService.deleteUserStory(savedUserStory.getId());
+
+        //then
+        Assert.assertNull("The user story should have been deleted", userStoryService.getUserStory(savedUserStory.getId()));
```

Comment:
When a record is not found in the database we throw an `EntityNotFoundException`. So, when we do a `userStoryService.getUserStory(savedUserStory.getId())` we will get a null. Instead an exception will be thrown. 

If we expect no record to be found, then we should expect an `EntityNotFoundException`, like we do in the test bellow, in the `userStoryNotFoundExceptionTest()`